scripts/schema/subset_filter.py

```diff
@@ -1,28 +1,26 @@
 import glob
 import yaml
+import copy
```

Comment:
Also here - did this `import` end up unneeded?

Reply:
Oops, missed this. Will remove.